tests/autoblocks/test_prompts_autogenerate.py

```diff
@@ -61,7 +61,7 @@ def test_infer_type():
     assert infer_type(0.0) == "Union[float, int]"
     assert infer_type(True) == "bool"
     assert infer_type(False) == "bool"
-    assert infer_type([""]) == "List[str]"
+    assert infer_type([""]) == "list[str]"
```

Comment:
Another understanding check - 

We were using `types.List` before because the built in list didn't support subscriping (like list[str]) until python 3.9 - now that we are dropping 3.9, we can just use `list`

Reply:
yeah exactly